pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1800,16 +1810,16 @@ func isDeviceOpened(deviceToDetach AttachedVolume, hostUtil hostutil.HostUtils)
 	return deviceOpened, nil
 }
 
-func useCSIPlugin(vpm *volume.VolumePluginMgr, spec *volume.Spec) bool {
+func useCSIPlugin(og OperationGenerator, spec *volume.Spec) bool {
```

Comment:
Alternatively, this func could consume a VolumePluginMgr and CSITranslator as explicit args, which narrows the set of things this func has access to.